mantidproject/mantid
# Use different colours for each fit curve on Quasi

Fixes [#11212](http://trac.mantidproject.org/mantid/ticket/11212).

To test:
- Open Indirect > Bayes > Quasi
- Load input `irs26176_graphite002_red.nxs`
- Load resolution `irs26173_graphite002_res.nxs`
- Run using both programs (should always get unique colours for fit and diff plots).
